requests_ratelimiter/requests_ratelimiter.py

```diff
@@ -72,6 +73,7 @@ def __init__(
         self.limit_statuses = limit_statuses
         self.max_delay = max_delay
         self.per_host = per_host
+        self.bucket = bucket
```

Comment:
Also, would you mind adding a simple unit test for this?

Reply:
Thanks for the quick response. The reason I implemented it the way I did was to make the behavior clearer.  having to set per-host AND bucket could be confusing. Setting bucket alone would have no effect and "fail" silently.

 I'll get a unit test done shortly.

Reply:
I've pushed the changes and the unit test. 

In the process, I identified an issue in that each session has its own rate-limiting so in the case of the unit test, setting bucket_name has no effect using the default in-memory rate limiting and calling the same URL from each session.

I've added a comment to the readme to this effect.

In this process, I have discovered that I do need to use the sqlite backend because I am using the requests library inside a multi-threaded web app (which I assume is a pretty common use case) and I had not realised that the rate limiting applies to each session independently.

I will make the change in my code base in the next few days and add a new PR to update the documentation with an example. It may even be worthwhile creating a shared-memory backed for this use case.